aten/src/ATen/native/Distance.cpp

```diff
@@ -271,71 +272,31 @@ Tensor _pdist_backward(const Tensor& grad, const Tensor& self, const double p, c
 }
 
 Tensor cosine_similarity(const Tensor& x1_, const Tensor& x2_, int64_t dim, double eps) {
-  /*
-   * cosine_similarity(x1, x2) = <x1, x2> / (||x1|| * ||x2||)
-   *
-   * The current implementation is an improvement over the previous version.
-   *
-   * Previous implementation:
```

Comment:
I think the comparison with the old implementation is still relevant in terms of numerical precision.
Also this comment is still valid for the updated implementation right?

Reply:
reverted